### ISSUE TITLE
[kapt] An exception occurred: java.lang.NoSuchFieldError: savedParameterNames

### ISSUE THREAD

**Issue by alatta** [facebook/litho#622]
- [ Y] I have searched [existing issues](https://github.com/facebook/litho/issues) and this is not a duplicate

## Version
lithoVersion : 0.11.0
```
    implementation "com.facebook.litho:litho-core:${lithoVersion}"
    implementation "com.facebook.litho:litho-widget:${lithoVersion}"
    compileOnly "com.facebook.litho:litho-annotations:${lithoVersion}"
    kapt "com.facebook.litho:litho-processor:${lithoVersion}"
    implementation 'com.facebook.soloader:soloader:0.2.0'
```

## Issues and Steps to Reproduce
```
[kapt] An exception occurred: java.lang.NoSuchFieldError: savedParameterNames 	at com.facebook.litho.specmodels.processor.MethodExtractorUtils.getSavedParameterNames(MethodExtractorUtils.java:87) 	at com.facebook.litho.specmodels.processor.MethodExtractorUtils.getMethodParams(MethodExtractorUtils.java:45) 	at com.facebook.litho.specmodels.processor.DelegateMethodExtractor.getDelegateMethods(DelegateMethodExtractor.java:65) 	at com.facebook.litho.sections.specmodels.processor.GroupSectionSpecModelFactory.createModel(GroupSectionSpecModelFactory.java:100) 	at com.facebook.litho.sections.specmodels.processor.GroupSectionSpecModelFactory.create(GroupSectionSpecModelFactory.java:90) 	at com.facebook.litho.specmodels.processor.AbstractComponentsProcessor.process(AbstractComponentsProcessor.java:66) 	at org.jetbrains.kotlin.kapt3.base.incremental.IncrementalProcessor.process(incrementalProcessors.kt) 	at org.jetbrains.kotlin.kapt3.base.ProcessorWrapper.process(annotationProcessing.kt:132) 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:794) 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:705) 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1800(JavacProcessingEnvironment.java:91) 	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1035) 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1176) 	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1170) 	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1068) 	at org.jetbrains.kotlin.kapt3.base.AnnotationProcessingKt.doAnnotationProcessing(annotationProcessing.kt:80) 	at org.jetbrains.kotlin.kapt3.base.AnnotationProcessingKt.doAnnotationProcessing$default(annotationProcessing.kt:36) 	at org.jetbrains.kotlin.kapt3.AbstractKapt3Extension.runAnnotationProcessing(Kapt3Extension.kt:223) 	at org.jetbrains.kotlin.kapt3.AbstractKapt3Extension.analysisCompleted(Kapt3Extension.kt:187) 	at org.jetbrains.kotlin.kapt3.ClasspathBasedKapt3Extension.analysisCompleted(Kapt3Extension.kt:98) 	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM$analyzeFilesWithJavaIntegration$2.invoke(TopDownAnalyzerFacadeForJVM.kt:95) 	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM.analyzeFilesWithJavaIntegration(TopDownAnalyzerFacadeForJVM.kt:105) 	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM.analyzeFilesWithJavaIntegration$default(TopDownAnalyzerFacadeForJVM.kt:80) 	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler$analyze$1.invoke(KotlinToJVMBytecodeCompiler.kt:398) 	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler$analyze$1.invoke(KotlinToJVMBytecodeCompiler.kt:65) 	at org.jetbrains.kotlin.cli.common.messages.AnalyzerWithCompilerReport.analyzeAndReport(AnalyzerWithCompilerReport.kt:107) 	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.analyze(KotlinToJVMBytecodeCompiler.kt:389) 	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.compileModules$cli(KotlinToJVMBytecodeCompiler.kt:118) 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:166) 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:56) 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:84) 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:42) 	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:104) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl$compile$1$1$2.invoke(CompileServiceImpl.kt:442) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl$compile$1$1$2.invoke(CompileServiceImpl.kt:102) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl$doCompile$$inlined$ifAlive$lambda$2.invoke(CompileServiceImpl.kt:1005) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl$doCompile$$inlined$ifAlive$lambda$2.invoke(CompileServiceImpl.kt:102) 	at org.jetbrains.kotlin.daemon.common.DummyProfiler.withMeasure(PerfUtils.kt:138) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl.checkedCompile(CompileServiceImpl.kt:1047) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl.doCompile(CompileServiceImpl.kt:1004) 	at org.jetbrains.kotlin.daemon.CompileServiceImpl.compile(CompileServiceImpl.kt:441) 	at sun.reflect.GeneratedMethodAccessor102.invoke(Unknown Source) 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) 	at java.lang.reflect.Method.invoke(Method.java:498) 	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:357) 	at sun.rmi.transport.Transport$1.run(Transport.java:200) 	at sun.rmi.transport.Transport$1.run(Transport.java:197) 	at java.security.AccessController.doPrivileged(Native Method) 	at sun.rmi.transport.Transport.serviceCall(Transport.java:196) 	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:573) 	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:834) 	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:688) 	at java.security.AccessController.doPrivileged(Native Method) 	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:687) 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) 	at java.lang.Thread.run(Thread.java:748) |  
-- | --
```
This started when I updated Android Studio to 3.5.3 and Kotlin plugin to 1.3.61. 
```
gradle_version = 5.1.1
gradle_plugin_version = 3.4.1
kotlin_version = 1.3.61
```

Please let me know if its a kapt issue.

**Comment by topwu** [comment]
I cannot reproduce the issue. I think you could either
- upgrade `lithoVersion` to the latest stable version(0.33.0) since 0.11.0 is quite old, or
- try to reproduce your issue in the sample app and provide us detailed repro steps.

Let us know if the approach can solve your issue or not.

**Comment by colriot** [comment]
Hey @alatta, were you able to reproduce this issue with the latest Litho version?

**Comment by alatta** [comment]
@colriot Since between 0.11.0 and 0.33.0 there were a lot of breaking changes and our app has extensive litho usage, updating won't be possible for us right now. To try the other approach I would still need to update Android studio to 3.5.3 and Kotlin to 1.3.61 which caused the issue originally with our app.

**Comment by colriot** [comment]
@alatta the fix to that was introduced in :https://github.com/facebook/litho/commit/5ba04ae5e1d94a9bd7c58d728f8bf75412751fc7 which was included in [v0.12.0](https://github.com/facebook/litho/releases/tag/v0.12.0). So, there is no need to update to the latest version, but only to `v0.12.0`. If it's still not possible, I suggest to apply this fix manually on your fork.

Closing this, feel free to reopen if anything pops up
